controllers/bjjControllers.js

```diff
@@ -0,0 +1,55 @@
+
+var db = require('../models');
+
+function index(req, res) {
+ 
+  db.Bjj.find({}, function(err, allAlbums) {
+    res.json(allAlbums);
+  });
+}
+
+
+// create function to create data in database
+function create(req, res) {
+  console.log('body', req.body);
+
+
+  db.Bjj.create(req.body, function(err, reviews) {
```

Comment:
This gives back a singular, not a plural, so should probably have the variable name `review` rather than `reviews`